Feat/#1198 knowledge base

AndrewDavydenko opened the pull request:
-   [x] Have you followed the [contributing guidelines](https://github.com/ever-co/gauzy/blob/master/.github/CONTRIBUTING.md)?
-   [x] Have you explained what your changes do, and why they add value?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

---
![Screen Shot 2020-07-08 at 5 55 01 PM](https://user-images.githubusercontent.com/20084357/86934657-9f228d00-c144-11ea-8d5b-0c3735b5813b.png)
updated design for sidebar(fast-fix)
added searching authors by article id 
creating authors for the article with bulk
deleting authors for the article with bulk when editing
but this feature has some issues so I commented call of them for successful builds, tomorrow will fix them

evereq commented:
@AndrewDavydenko ok, no problem, we can merge it tomorrow, after your fix, thanks!

AndrewDavydenko commented:
09.07.2020 
fixed issues with adding&removing authors for article
small UI improve for adding/editing article
in progress displaying author on the article card
in feature - delete author when deleting articles, categories or bases

evereq commented:
@AndrewDavydenko I still see lots of code commented out......

AndrewDavydenko commented:
10.07.2020 
added authors to the article card
fixed bug with enormous authors choosing
added UI for searching by name&author
the logic for searching in progress
